sunpy/map/mapbase.py

```diff
@@ -317,6 +323,26 @@ def _repr_html_(self):
         count_nan = np.isnan(self.data).sum()
         count_inf = np.isinf(self.data).sum()
 
+        # If data is from a dask array, pull data into memory for quicklook() method
+        # For basic html representation, only use dask html representation without computing data
+        if DASK_INSTALLED and isinstance(finite_data, DaskArray):
+            if compute_dask:
+                finite_data = finite_data.compute()
```

Comment:
```suggestion
            # This will fetch the entire data array into memory.
            if compute_dask:
                finite_data = finite_data.compute()
```

Reply:
made changes as requested